src/Microsoft.AspNet.Hosting/Logging/RequestInformationValues.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Framework.Logging;
+
+namespace Microsoft.AspNet.Hosting
+{
+    public class RequestInformationValues : ILogValues
+    {
+        public RequestInformationValues(Guid requestId)
+        {
+            RequestId = requestId;
+        }
+
+        public Guid RequestId { get; }
+
+        /// <inheritdoc />
+        public string Format()
+        {
+            return LogFormatter.FormatLogValues(this);
+        }
+
+        /// <inheritdoc />
+        public IEnumerable<KeyValuePair<string, object>> GetValues()
+        {
+            return new[] 
```

Comment:
yield return new KeyValuePair<string, object>(nameof(RequestId), RequestId); // instead of a new[]?